x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/calculate_rule_source_for_import.ts

```diff
@@ -26,31 +27,44 @@ import { convertRuleToImportToRuleResponse } from './converters/convert_rule_to_
  * @returns The calculated rule_source and immutable fields for the rule
  */
 export const calculateRuleSourceForImport = ({
-  rule,
+  importedRule,
```

Comment:
nit: It doesn't look like a has been imported when this function is invoked. A name like `ruleToImport` or `beingImportedRule` feels like resonating better.